source/renderer/app/api/ada/types.js

```diff
@@ -175,7 +175,15 @@ export const AdaV1AssuranceOptions: {
   NORMAL: 'normal', STRICT: 'strict',
 };
 
-export type AdaTransactionsV1 = Array<AdaTransactionV1>;
+export type AdaTransactionsV1 = {
+  data: Array<AdaTransactionV1>,
+  meta: {
+    pagination: {
+      totalPages: number,
```

Comment:
@daniloprates please add the types for all other possible meta data props from the docs

Reply:
I’ve got a type for meta data. You can get the entirety as BaseResponseV1 or Pagination if you just want that.

Reply:
@MarcusHurney great. @nikolaglumac done.